var/spack/repos/builtin/packages/gcc/package.py

```diff
@@ -70,8 +67,7 @@ class Gcc(AutotoolsPackage):
     depends_on("gmp")
     depends_on("mpc", when='@4.5:')
     depends_on("isl", when='@5.0:')
-    depends_on("binutils~libiberty", when='+binutils ~gold')
-    depends_on("binutils~libiberty+gold", when='+binutils +gold')
+    depends_on("binutils~libiberty", when='+binutils')
```

Comment:
We've decided that individual packages should not need to depend on `binutils`, that the Spack user is responsible for supplying a recent version.  However, *not* depending on `binutils` in this case could make life unpleasant for the user trying to bootstrap Spack.